### PR TITLE
Add context menu to Symphony Electron

### PR DESCRIPTION
**Pull request by rbarros-daitan**
Electron-38

### CODE REVIEW

File: js/main.js [hunk 1]
```diff
@@ -47,6 +51,8 @@ function getUrlAndOpenMainWindow() {
         let title = 'Error loading configuration';
         electron.dialog.showErrorBox(title, title + ': ' + err);            
     });
+
+    applyContextMenu();
```
Comment (lneir) [inline js/main.js]: can we instead move this into windowMgr.js code and put in createMainWindow and createChildWindow and then pass window object in via opts parameter...e.g., applyContextMenu({ window: mainWindow});  and plyContextMenu({ window: childWindow});

File: js/contextMenu.js [hunk 2]
```diff
@@ -0,0 +1,48 @@
+'use strict';
```
Comment (lneir) [inline js/contextMenu.js]: also you could create a subdir for the code: js/menus and move both this and menuTemplate.js into that dir.

File: js/main.js (hunk 1 above)
Comment (rbarros-daitan) [inline js/main.js]: Do we plan to have different right click menu based on child or main window?
Current implementation applies the same context menu on main window and on any window created after that.

File: js/contextMenu.js (hunk 2 above)
Comment (rbarros-daitan) [inline js/contextMenu.js]: ok, done.

File: js/main.js (hunk 1 above)
Comment (lneir) [inline js/main.js]: might be diff but for now the same.  also want to move code into here since this involves creating windows.

**Pull request merged**